tfe/resource_tfe_workspace_test.go

```diff
@@ -759,6 +761,66 @@ func TestAccTFEWorkspace_updateVCSRepo(t *testing.T) {
 	})
 }
 
+func TestAccTFEWorkspace_updateVCSRepoTagsRegex(t *testing.T) {
+	workspace := &tfe.Workspace{}
+	rInt := rand.New(rand.NewSource(time.Now().UnixNano())).Int()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck: func() {
+			testAccPreCheck(t)
+			if GITHUB_TOKEN == "" {
```

Comment:
This is third time these checks are repeated in this file, IMHO it warrants method extraction